meta-multimedia/recipes-multimedia/libcamera/libcamera.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Linux libcamera framework"
+SECTION = "libs"
+
+LICENSE = "GPL-2.0 & LGPL-2.1"
+
+LIC_FILES_CHKSUM = "\
+    file://licenses/gnu-gpl-2.0.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+    file://licenses/gnu-lgpl-2.1.txt;md5=4b54a1fd55a448865a0b32d41598759d \
+"
+
+SRC_URI = "git://linuxtv.org/libcamera.git;protocol=git"
+SRCREV = "53eab996809e826bc914e4c34c78fe74d86f8dc4"
+
+PV = "201910+git${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = "python3-native python3-pyyaml-native udev"
+
+inherit meson pkgconfig
```

Comment:
I think you need to also inherit python3native

Reply:
and you dont then need DEPENDS on python3-native